zdm/parameters.py

```diff
@@ -143,18 +136,58 @@ class HostParams(data_class.myDataClass):
 # FRB intrinsic width parameters
 @dataclass
 class WidthParams(data_class.myDataClass):
-    logmean: float = field(
+    Wlogmean: float = field(
         default = 1.70267, 
         metadata={'help': 'Intrinsic width log of mean',
                   'unit': 'ms',
                   'Notation': '\mu_{w}',
                   })
-    logsigma: float = field(
+    Wlogsigma: float = field(
         default = 0.899148,
         metadata={'help': 'Intrinsic width log of sigma',
                   'unit': 'ms',
                   'Notation': '\sigma_{w}',
                   })
+    Wthresh: int = field(
+        default=0.5,
+        metadata={'help': 'Starting fraction of intrinsic width for histogramming'})
```

Comment:
please add the `unit` for each of these.

Reply:
Done. All without units are unitless, but have added empty field of ''